utils/nettest/main.go

```diff
@@ -0,0 +1,32 @@
+// Package nettest provides a modified version of the GetFreeTCPPorts function
+// from Teleport's utils package.
+package nettest
+
+import (
+	"net"
+	"strconv"
+
+	utils "github.com/gravitational/teleport/lib/utils"
+	"github.com/gravitational/trace"
+)
+
+// GetFreeTCPPortsForTests returns n free ports (which are suggested by the kernel)
+func GetFreeTCPPortsForTests(n int) (utils.PortList, error) {
```

Comment:
you could revert this and related changes, up to you

Reply:
I'll leave it in.